Add TeamMembersGrid

chrismekim opened the pull request:
### Description

<!--- Summarize the changes that can be found in this PR and how your reviewers should test these changes. -->
- Adds TeamMembersGrid to `/about`
- Adds `PersonCard` (this component will be used for Founders and Team Members)

### Type(s) of changes

<!--- Put an `x` in all the boxes that apply. -->

- [x] New feature

### Motivation for PR

<!--- If it addresses an open issue, please link to the issue here, otherwise, briefly describe the issue. -->

### How Has This Been Tested?

<!--- Please note how you have tested your changes. Browsers, accessibility, devices, unit tests, etc. -->
- View at `/about`
### Applicable screenshots:

<!--- When appropriate, upload screenshots. -->
![image](https://user-images.githubusercontent.com/43737723/100290038-fb454c80-2f3f-11eb-9276-4b211ff6c9e1.png)

### Follow-up PR

<!--- When appropriate, please note what your reviewers can expect in a follow up PR. -->

limsohee1002 reviewed (commented):
On preview, outside of social platform link is clickable. Also, Instagram and Twitter gap seem larger than the Linkedin and Twitter.

chrismekim commented:
> On preview, outside of social platform link is clickable. Also, Instagram and Twitter gap seem larger than the Linkedin and Twitter.

Fixed and updated the styling to use a margin right instead of using columns to give width, since I think in the design they are not actually even columns like I had created!
